scout/server/blueprints/cases/templates/cases/causatives.html

```diff
@@ -1,7 +1,7 @@
 {% extends "layout_bs4.html" %}
 
 {% block title %}
-  {{ super() }} - Institutes
+{{ super() }} - Institutes
 {% endblock %}
 
```

Comment:
```suggestion
{% block css %}
{{ super() }}
  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.20/css/jquery.dataTables.min.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css">
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/1.6.0/css/buttons.dataTables.min.css">
{% endblock %}
```

Reply:
What is the {{ super() }} doing?!




{{ super() }}
 

Reply:
super() is allowing this file to import all the eventual code present in the {% block css %} of the template that is extended in this file: which is scout/scout/server/templates/bootstrap4.html. 
In this case there is no css block in layout_bs4.html, but since layout_bs4.html is extending bootstrap4.html then this file is indirectly extending bootstrap4.html

Reply:
Aha! Sorry for stupid question, but where is %block css% defined? I mean there is no template with that name. How does this html find layout_bs4.html?

Reply:
First line of this template is: {% extends "layout_bs4.html" %}
which means you are importing everything which is in layout_bs4.html.

First line of layout_bs4.html is {% extends "bootstrap4.html" %} which means layout_bs4.html is importing all the code from bootstrap4.html.

This means that this template in turn import everything from bootstrap4.html